zproject/test_extra_settings.py

```diff
@@ -270,3 +270,9 @@ def set_loglevel(logger_name: str, level: str) -> None:
 }
 
 ALLOW_ANONYMOUS_GROUP_VALUED_SETTINGS = True
+
+# This override disables the grace period for undoing resolving/unresolving
+# a topic in tests.
+# This allows tests to not worry about the special behavior during the grace period.
+# Otherwise they would have to do lots of mocking of the timer to work around this.
+RESOLVE_TOPIC_UNDO_GRACE_PERIOD_SECONDS = 0
```

Comment:
We usually want to keep tests behavior same as in production, but it feels like making regular "resolve topic" test have to think about this adds a bunch of annoying overhead, so I figured it might be good to disable it? Should be easy to remove this and just fix up the tests if we think that's better though.